Fix ledger test brent method

## Description
FAILED tests/syft/core/adp/data_subject_ledger_test.py::test_cache_indexing_correctness - ValueError: Use of `bounds` is incompatible with 'method=Brent'.
FAILED tests/syft/core/adp/data_subject_ledger_test.py::test_cache_bypass - ValueError: Use of `bounds` is incompatible with 'method=Brent'.

## How to Reproduce
See CI:
https://github.com/OpenMined/PySyft/actions/runs/3842992045/jobs/6544847611
`pytest -s -m 'fast or slow' --cov syft --cov-fail-under 65 -n auto --durations=50`

--- a/packages/syft/src/syft/core/adp/data_subject_ledger.py
+++ b/packages/syft/src/syft/core/adp/data_subject_ledger.py
@@ -335,7 +335,7 @@ class DataSubjectLedger(AbstractDataSubjectLedger):
         f = self._get_fake_rdp_func(constant=constant)
         f2 = self._get_alpha_search_function(rdp_compose_func=f)
         results = minimize_scalar(
-            f2, method="Brent", bracket=(1, 2), bounds=[1, np.inf]
+            f2, method="Brent", bracket=(1, 2)#, bounds=[1, np.inf]
         )
 
         return results.x, results.fun

